packages/react/src/components/Video/IVideoProps.ts

```diff
@@ -0,0 +1,22 @@
+import { Video } from '@signalwire/js';
+import { RefObject } from 'react';
+type RoomSessionOptions = ConstructorParameters<Video.RoomSession>[0];
+
+export interface IVideoProps extends Omit<RoomSessionOptions, 'rootElement'> {
+  rootElement?: RefObject<HTMLElement>;
```

Comment:
should `rootElement` only be in `ICoreVideoProps`?

Reply:
Yup. The <Video/> element should automatically manage the rootElement stuff (for the web, at least)

Reply:
Yup. The `rootElement` should be managed by the Video component (for the web at least)